Configuration/TCA/tx_books_domain_model_books.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+return [
+    'ctrl' => [
+        'label' => 'title',
+        'title' => 'LLL:EXT:books/Resources/Private/Language/Tca.xlf:books',
```

Comment:
AFAIK, the convention is to use `locallang_db.xlf` for the TCA labels.

Reply:
Seems to be split between `locallang_tca.xlf` and `locallang_db.xlf`, but IMO the `locallang_` prefix is unnecessary, since the files are in their own folder nowadays (they were not previously).